test/apiv2/python/rest_api/test_v2_0_0_container.py

```diff
@@ -14,6 +14,10 @@ def test_list(self):
         obj = r.json()
         self.assertEqual(len(obj), 1)
 
+    def test_list_filters(self):
+        r = requests.get(self.podman_url + "/v1.40/containers/json?filters={'status':'running'}")
```

Comment:
Any chance we can check the output of the query, see that there are (or are not?) running containers?